pramen/core/src/main/scala/za/co/absa/pramen/core/runner/AppRunner.scala

```diff
@@ -202,7 +202,7 @@ object AppRunner {
           throw new IllegalArgumentException(s"Non-existent or disabled jobs selected for execution. Output tables: ${notFoundJobs.mkString(", ")}")
         }
 
-        jobs.filter(job => selectedTablesSet.contains(job.outputTable.name))
+        jobs.filter(job => selectedTablesSet.contains(job.outputTable.name) || job.outputTable.format.isTransient)
```

Comment:
Does this not have to be `isLazy`?

Reply:
I thought about it. So this part of the code selects which jobs to execute when a user selects individual jobs instead of running the full pipeline. I guess yes, for 'transient_eager' to be executed, the user should either execute the full pipeline or explicitly select it.